ext/web/13_message_port.js

```diff
@@ -160,9 +171,25 @@ class MessagePort extends EventTarget {
           );
         } catch (err) {
           if (ObjectPrototypeIsPrototypeOf(InterruptedPrototype, err)) break;
+          if (
+            typeof this[workerThreadCloseCb] == "function" &&
+            !this[_workerThreadCloseCbInvoked]
+          ) {
+            this[workerThreadCloseCb]();
+            this[_workerThreadCloseCbInvoked] = true;
+          }
```

Comment:
How about this move it to a top-level helper like: `nodeWorkerThreadMaybeInvokeCloseCb`?